coin-vester-front/src/utils/write-vesting-session.tsx

```diff
@@ -75,6 +75,10 @@ export function useWriteVestingSession(client?: Client) {
             variant="error"
           />,
         );
+      })
+      .finally(() => {
+        setIsPending(false);
+        setOpId(undefined);
```

Comment:
when pending if false, either success of error is true, it means I can't see the op id, because you pass it here at undefined